resources/components/base/Button.vue

```diff
@@ -27,12 +27,10 @@ module.exports = {
 
 	props: {
 		disabled: {
-			type: Boolean,
-			default: false
+			type: Boolean
 		},
-		framed: {
-			type: Boolean,
-			default: true
+		unframed: {
```

Comment:
This isn't my favorite thing, but I think it makes sense for boolean vars to default to undefined, and then devs can set them to true if needed. In the case of framed/unframed the language gets a bit confusing, but I think it's worth it for the sake of consistency.

Reply:
What about `frameless` instead of `unframed`? I think it's a little less awkward to say `frameless: true`

Reply:
Yessss perfect, will make that change